src/app_engine/analytics_page.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+"""Module for the AnalyticsPage handler."""
+
+import json
+import time
+
+import webapp2
+
+import analytics
+import constants
+from constants import AnalyticsField
+
+
+class AnalyticsPage(webapp2.RequestHandler):
+  """Client Analytics data handler.
+
+  Each POST body to the AnalyticsPage is a JSON object of the form,
+  {
+    'request_time_ms': <client time in milliseconds when the request was sent>,
+    'type': <type of the content below>,
+    'content': <content related to the above type>
+  }
+
+  Currently the AnalyticsPage handles the following types:
+  'event': A JSON event object of the form:
+          {
+            'event_type': <string corresponding to an attribute of EventType>,
+            'event_time_ms': <client time when the event occurred>,
+            'room_id': <id of the room corresponding to the event [optional]>,
+          }
+
+      The event is reported to Analytics.report_event with the
```

Comment:
Can you take a look at the new wording. I reworked how the JSON request is sent as well to make it more "proto" like based on your other comments below.